File: src/main/java/com/hzj/myblog/service/BlogTypeAccountService.java
package com.hzj.myblog.service;

import com.hzj.myblog.model.BlogTypeAccount;

import java.util.List;

/**
 * 博客分类统计service接口
 *
 * @author hzj
 */
public interface BlogTypeAccountService {
    /**
     * 查找博客分类数量
     *
     * @return 博客数量列表
     */
    List<BlogTypeAccount> findUserBlogTypeAccount(Integer userId);
}

File: my-blog-web/src/app/@core/interface/blog-type-account.service.ts
import {Observable} from "rxjs";
import {ReturnModel} from "../../entity/return-model";
import {BlogTypeAccount} from "../../model/blog-type-account";

export abstract class BlogTypeAccountService {

  /**
   * 查找博客分类数量
   *
   * @return 博客数量列表
   */
  abstract findUserBlogTypeAccount(userId: number): Observable<ReturnModel<BlogTypeAccount[]>>;
}
